src/core/core.controller.js

```diff
@@ -1092,36 +1092,33 @@ class Chart {
 		// - it would be expensive.
 		const useFinalPosition = replay;
 
+		let active = [];
 		let changed = false;
 
 		// Find Active Elements for hover and tooltips
 		if (e.type === 'mouseout') {
-			me.active = [];
+			active = [];
```

Comment:
can be removed? (already initialized to [])

Reply:
Removed